controllers/userController.js

```diff
@@ -13,12 +15,47 @@ const createUser = async (req, res) => {
     const savedUser = await newUser.save();
 
     res.status(201).json(savedUser);
+
+    const token = jwt.sign({userId: user._id}, "your-secret-key", {
+      expiresIn: "1h"
+    })
+    
+    res.status(200).json({ message: "Login successful", user, token });
```

Comment:
mislim da ce se ovako vracat cili user s ID, to nije podatak koji bi trebali slat dalje